Content.Server/DiscordAuth/DiscordAuthManager.cs

```diff
@@ -0,0 +1,129 @@
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Content.Shared.CCVar;
+using Content.Shared.DiscordAuth;
+using JetBrains.Annotations;
+using Robust.Server.Player;
+using Robust.Shared.Configuration;
+using Robust.Shared.Enums;
+using Robust.Shared.Network;
+using Robust.Shared.Player;
+
+namespace Content.Server.DiscordAuth;
+
+// TODO: Add minimal Discord account age check for panic bunker by extracting timestamp from snowflake received from API secured with key
```

Comment:
This would check the age of the discord account?

If so: why?

If not: what does this do?

Not to seem rude :(

Reply:
Prevent raiders from creating brand new accounts and joining immediately after being banned.